src/EA.Iws.RequestHandlers/NotificationMovements/BulkReceiptRecovery/ReceiptRecoveryReceiptOnlyRule.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace EA.Iws.RequestHandlers.NotificationMovements.BulkReceiptRecovery
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Core.Movement;
+    using Core.Movement.BulkReceiptRecovery;
+    using Core.Rules;
+    using Domain.Movement;
+
+    public class ReceiptRecoveryReceiptOnlyRule : IReceiptRecoveryContentRule
+    {
+        private readonly IMovementRepository repo;
+
+        public ReceiptRecoveryReceiptOnlyRule(IMovementRepository repo)
+        {
+            this.repo = repo;
+        }
+
+        public async Task<ReceiptRecoveryContentRuleResult<ReceiptRecoveryContentRules>> GetResult(List<ReceiptRecoveryMovement> movements, Guid notificationId)
+        {
+            var actualMovements = await repo.GetAllMovements(notificationId);
+
+            List<int> shipments = new List<int>();
+            MessageLevel result = MessageLevel.Success;
+
+            foreach (var movement in movements.Where(p => p.MissingRecoveredDisposedDate))
+            {
+                var actualMovement = actualMovements.FirstOrDefault(p => p.Number == movement.ShipmentNumber);
+
+                if (actualMovement.Status == MovementStatus.Captured)
```

Comment:
Please see my comments on `ReceiptRecoveryShipmentMustBePrenotifiedRule.cs`. The conditions seems to be same here too.